README.md

```diff
@@ -1,69 +1,378 @@
-# owlet-ui
+# Owlet UI
 
 A [re-frame](https://github.com/Day8/re-frame) application for the Owlet Project.
 
-## Development Mode
+## Development
 
-### Compile css
+> Henceforth, for clarity and accuracy, we'll assume you're on MacOS. The use
+> of other platforms with these tools is certainly possible, and the process is
+> probably quite similar to the exposition below, but it is currently untested.
 
-Compile css file once.
+### Installation
+
+- Make sure you have a recent version of the Java Development Kit. If not,
+  download the installer from [the Oracle downloads page](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html).
+  Select **Accept License Agreement** at the top of the panel for the
```

Comment:
Yes, that was one of the hardest parts. It's unbelievable how hard Oracle makes it to find where to go to download stuff! A newbie would be understandably turned off to Clojure because of this.

Merging.